### DISCUSSION TITLE
Transfer incomplete

### DISCUSSION THREAD

**Discussion by Smiley006** [iterate-ch/cyberduck#15190] [Q&A]
I’m trying to back up to Google. It states that there is something like 45 GB to back up, but it usually stops at 0, or just a couple of gigabytes before it just seems to freeze and locks up. I’ve been trying for weeks to get this to complete the backup, unsuccessfully. I have a number of other backups to Microsoft and Google which are working successfully, although they are much smaller quantities, so I can’t explain why this would always fail. Running Mac Ventura, 13.6. Thanks for your help.

**Comment by tatze96** [comment]
Does the issue persist within the current Cyberduck version (8.7.1)?

If so, can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issue, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15190 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Comment by Smiley006** [comment]
Attachment available until Dec 12, 2023
Hello and thank you for your email.
Yes, I'm using 8.7.1 and it does persist.
I have attached the log file.
Thanks very much for your help.
Take care,
Harry

Click to Download
 <https://www.icloud.com/attachment/?u=https%3A%2F%2Fcvws.icloud-content.com%2FB%2FAebvHcOuUlW0Gu9uX8CGnSodpwgYATqIqJ6pQK5xsM6Nw0eOpaQFIcn_%2F%24%7Bf%7D%3Fo%3DAl3bF7ZpwydIfyjIYYrh05bonBtKK7aALSJF_J5PwMU7%26v%3D1%26x%3D3%26a%3DCAog_HsYHuqlWnsDhNi_Fmr3ccJYJR7jdoJtOeH-WeofnqsSdhDChPapvDEYwpTx_cUxIgEAKgkC6AMA_w2YSQdSBB2nCBhaBAUhyf9qJUE9kwWFja2GVDN9HwL-4axsc9xKQKQ4O1uzJ3QNJFYMfSlP8JxyJZ4xjbO5SOyCRYKBuV5XyXyG2lCMtolqtFmWnTuUzFO8AG-CZdg%26e%3D1702413224%26fl%3D%26r%3D22DE3B15-0911-4B6F-B2C2-DD199BEAAA8C-1%26k%3D%24%7Buk%7D%26ckc%3Dcom.apple.largeattachment%26ckz%3DAF271D1A-85C8-407E-A518-ED7457039999%26p%3D105%26s%3D0qio9uYbQpyr4_1vPe2DrPDhAWo&uk=jbv_V5I4PizC0AyBpIQjSw&f=cyberduck.log&sz=27208174>cyberduck.log
27.2 MB

**Comment by dkocher** [comment]
In general bulk file transfers to _Google Drive_ are expected to be slow and we would recommend Google Cloud Storage instead to use for archiving.
